blazar_dashboard/content/leases/templates/leases/calendar.html

```diff
@@ -3,7 +3,7 @@
 {% block title %}{% trans "Leases" %}{% endblock %}
 
 {% block page_header %}
-  {% include "horizon/common/_page_header.html" with title=_("Host Calendar") %}
+{% include "horizon/common/_page_header.html" with title=resource_type|add:" Calendar"|title %}
```

Comment:
clever little hack ;) -- we may want to not do the fancy part though for i18n reasons. Typically i18n stuff works by statically analyzing the code to find all the strings located with _(...) and then shoving them to translation services and generating language lookups using the text as a key. Dynamic text makes that harder. The odds of anybody translating the blazar calendar to portugese are admittedly slim, but it's good to be consistent :)